### DISCUSSION TITLE
FR translation for companion

### DISCUSSION THREAD

**Discussion by Pat687426** [EdgeTX/edgetx#4705] [General]
I've made some translations for companion.
If its OK, i will made some other in the future.
Patrick
[translation Fr companion.zip](https://github.com/EdgeTX/edgetx/files/14545996/translation.Fr.companion.zip)

**Comment by pfeerick** [comment]
~~PR with this is here => https://github.com/EdgeTX/edgetx/pull/4719~~

edit: no, it's part of https://github.com/EdgeTX/edgetx/pull/4613 now. When doing translations, it may be an idea to download a copy of Qt Linguist - i.e. from https://github.com/lelegard/qtlinguist-installers/releases - as it will ensure there are no mistakes in the tags/formatting, and will make it easy to see which translations are unfinished / still need doing ;)
